artemis-core-client/src/main/java/org/apache/activemq/artemis/utils/XMLUtil.java

```diff
@@ -52,6 +52,12 @@ private XMLUtil() {
       // Utility class
    }
 
+   public static String CONSIDER_OS_ENV_PROP = "org.apache.activemq.artemis.utils.considerOsEnv";
```

Comment:
An alternative name could be `REPLACE_OS_ENV_PROP`

Reply:
it does not replace tho, it only considers if the system prop is absent